docs/templates/runtime.adoc

```diff
@@ -11,20 +10,13 @@ Freeform description of runtime purpose
 
 // List of pallets with their config description
 
-=== `++pallet_name++` link:https://google.com[{github-icon},role=heading-link]
-
+=== Pallet Category
+[%collapsible]
+====
+===== `++pallet_name++` link:https://google.com[{github-icon},role=heading-link]
 Freeform description of the reason why this pallet was added.
+====
 
-```rust
-impl pallet_name::Config for Runtime {
-    type Config1 = Type1;
-    type Config2 = Type2;
-}
-```
-
-* Description of `Type1` if needed
-* Description of `Type2` if needed
```

Comment:
I still feel that this is needed in some form. Could you please include that freeform description may include important configuration parameters?